arquillian/adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/UberjarSimpleContainer.java

```diff
@@ -152,7 +153,8 @@ public void start(Archive<?> archive) throws Exception {
         final ShrinkwrapArtifactResolvingHelper resolvingHelper = ShrinkwrapArtifactResolvingHelper.defaultInstance();
 
         BuildTool tool = new BuildTool(resolvingHelper)
-                .fractionDetectionMode(BuildTool.FractionDetectionMode.never)
+                .fractionDetectionMode(BuildTool.FractionDetectionMode.when_missing)
```

Comment:
For running Java EE 7 Samples with Swarm, I sometimes had to add some fractions manually, because they couldn't be autodetected (see SWARM-974), but I only wanted to do minimal changes, so I used fraction autodetection mode of `force`.

It can easily be made configurable in `arquillian.xml`, see https://github.com/Ladicek/wildfly-swarm/commits/arquillian-fraction-autodetection and https://github.com/Ladicek/javaee7-samples/tree/wildfly-swarm